components/Deck/ContentPanel/SlideModes/SlideEditPanel/SlideContentEditor.js

```diff
@@ -959,6 +958,75 @@ class SlideContentEditor extends React.Component {
+            }
+            else {
+                questionhtml += '<div style="padding-bottom:15px;"><div>' + currentQuestion.title + '</div><ul  style="padding-bottom:5px; padding-top:5px;">';
+            }
+
+            if(showAnsExp){
+                //if the answers and explanation should be shown for the embedded questions
+
+                for (let j = 0; j < currentAnswers.length; j++){
+                    let correctText = '';
+
+                    switch (currentAnswers[j].correct) {
+                        case true:
+                            correctText = '<strong> - correct </strong>';
+                            break;
+                    }
+                    questionhtml += '<li style="font-size:22px; margin-top:5px;">'+ currentAnswers[j].answer + correctText + '</li>';
+                }
+                let explanation = currentQuestion.explanation ? '<div style="padding-left:30px; padding-bottom:5px; font-size:24px; "><strong>Explanation:</strong> '+ currentQuestion.explanation + '</div>' : '';
+                questionhtml += '</ul>'+ explanation +'</div>';
+            }
+            else {
+                //if the answers and explanation shouldn't be included
+                for (let j = 0; j < currentAnswers.length; j++){
+                    questionhtml += '<li style="font-size:22px; margin-top:5px;">'+currentAnswers[j].answer + '</li>';
+                }
+
+                questionhtml += '</ul></div>';
+            }
+        }
+        questionhtml += '</div>';
+
+        //let scrolldiv = '<div _type="body" id="questions_content" class="block content v-up context-menu-disabled" style="overflow-y:auto; height:80%; max-height:800px; position: relative; top: 15%; left: 40px; ">'+questionhtml+'</div>';
+        //let iframe = '<div class="iframe" style="position: absolute; top: 100px; left:80px; "><iframe width="800" height="550" srcdoc="'+ questionhtml + '" frameborder="0"></iframe></div>';
+        //let pptx2htmlDiv = '<div class="pptx2html" style="position: relative; width: 960px; height: 720px;">'+titleDiv + iframe+'</div>';
+        let pptx2htmlDiv = '<div class="pptx2html" style="position: relative; width: 960px; height: 720px;">'+titleDiv + questionhtml+'</div>';
+        
+        if($('.pptx2html').length) //if slide is in canvas mode
+        {
+            /*$('.pptx2html').append('<div id="'+uniqueID+'" style="position: absolute; top: 150px; left: 50px; width: 700px; height: 500px; z-index: '+(this.getHighestZIndex() + 10)+';">'+iframe+'</div>');
+            this.hasChanges = true;
+            //this.correctDimensionsBoxes('iframe'); */
+            this.refs.inlineContent.innerHTML = pptx2htmlDiv;
+        
+        } else { //if slide is in non-canvas mode
+            this.refs.inlineContent.innerHTML += scrolldiv; //does this want += or should it just be =? how does a canvas mode slide even work...
```

Comment:
This results to an error, because scrolldiv is not defined. Merging from master did not help. @kadevgraaf please review this as I'm lost here

Reply:
Looking into this now for the non-canvas mode

Reply:
I've pushed some changes. I think a line was accidentally commented out. I don't know much about non-canvas mode, but it definitely puts the questions in the slide now.